cordformation/src/test/resources/external-service-2.gradle

```diff
@@ -0,0 +1,5 @@
+ext {
+    externalServiceConfig = [
+            containerImage: "docker/test/customTomcat"
+    ]
+}
```

Comment:
You don't need this file.You can apply this external configuration _directly_ into `DeployTwoNodeCordappWithExternalServiceNoOption.gradle`